target/linux/ramips/image/mt7621.mk

```diff
@@ -408,6 +408,19 @@ define Device/beeline_smartbox-turbo
 endef
 TARGET_DEVICES += beeline_smartbox-turbo
 
+define Device/beeline_smartbox-turbo-plus
+  $(Device/sercomm_cxx)
+  IMAGE_SIZE := 32768k
+  SERCOMM_HWID := CQR
+  SERCOMM_HWVER := 10000
+  SERCOMM_SWVER := 2010
+  DEVICE_VENDOR := Beeline
+  DEVICE_MODEL := SmartBox TURBO+
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615-firmware kmod-usb3 \
+    uboot-envtools
```

Comment:
`uboot-envtools` can be removed as it's a default package now.

Reply:
Done.

Reply:
`Tested-by: Oleg Nabatchikov <2scorpio@inbox.ru>`